app/views/books/_form.html.haml

```diff
@@ -1,4 +1,4 @@
-= simple_form_for @book do |f|
+= simple_form_for [@user, @book] do |f|
```

Comment:
I would be consitent in where you get your user (current_user, @book.user, @user)

Reply:
agree, done ✅